### PR TITLE
packaging: changing the name to include ubuntu version

### PR DESCRIPTION
**Pull request by GregMeno**
@dmick 
@ktdreyer 

packages from different discributions cannot exist in the
same repository without this name component

Signed-off-by: Gregory Meno <gmeno@redhat.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 VERSION ?= $(shell ./get-versions.sh VERSION)
 REVISION ?= $(shell ./get-versions.sh REVISION)
 DIST ?= unstable
+DIST_VERSION ?= "~"$(shell sed '/CODENAME/!d;s/^.*=//' /etc/lsb-release)"1"
```
Comment (dmick) [inline Makefile]: lsb_release -sc

Comment (dmick) [inline Makefile]: And why not call it codename?  And does this work for rhel/Centos/Debian?

Comment (alfredodeza) [inline Makefile]: is there any place where the system understands that lsb-release is a dependency to parse this? lsb-release is not included by default everywhere

Comment (dmick) [inline Makefile]: Yes, lsb_release (the command) is required for calamari; if it's not explicit it should be

**Comment by GregMeno** [comment]
@alfredodeza https://github.com/ceph/calamari/blob/master/debian/control#L12

**Comment by GregMeno** [comment]
and https://github.com/ceph/calamari/blob/master/calamari.spec#L26

**Comment by ktdreyer** [comment]
> "debian version" is <version>-1<codename>[~<backport>], which splits at the - into <version> and <codename>[~<backport>], which I think is rightest.

After a discussion in IRC, we are agreed, and let's proceed with this convention (copying Ceph).

**Comment by GregMeno** [comment]
@dmick @ktdreyer we are leaving -$(REVISION) as part of version. Does that sound right?

**Comment by ktdreyer** [comment]
> @dmick @ktdreyer we are leaving -$(REVISION) as part of version. Does that sound right?
Sounds right to me

**Comment by GregMeno** [comment]
I want to merge it to a different base. I'll open a second PR

**Pull request closed without merging**